fabric-data-attachment-api-v1/src/testmod/java/net/fabricmc/fabric/test/attachment/AttachmentTestMod.java

```diff
@@ -57,6 +66,24 @@ public class AttachmentTestMod implements ModInitializer {
 	public static final AttachmentType<String> FEATURE_ATTACHMENT = AttachmentRegistry.create(
 			Identifier.of(MOD_ID, "feature")
 	);
+	public static final AttachmentType<Boolean> SYNCED_WITH_ALL = AttachmentRegistry.<Boolean>builder()
+			.initializer(() -> false)
+			.persistent(Codec.BOOL)
+			.syncWith(PacketCodecs.BOOL.cast(), AttachmentSyncPredicate.all())
+			.buildAndRegister(Identifier.of(MOD_ID, "synced_all"));
+	public static final AttachmentType<Boolean> SYNCED_WITH_TARGET = AttachmentRegistry.<Boolean>builder()
+			.initializer(() -> false)
+			.syncWith(PacketCodecs.BOOL.cast(), AttachmentSyncPredicate.targetOnly())
+			.buildAndRegister(Identifier.of(MOD_ID, "synced_target"));
+	public static final AttachmentType<Boolean> SYNCED_EXCEPT_TARGET = AttachmentRegistry.<Boolean>builder()
+			.initializer(() -> false)
+			.syncWith(PacketCodecs.BOOL.cast(), AttachmentSyncPredicate.allButTarget())
+			.buildAndRegister(Identifier.of(MOD_ID, "synced_expect_target"));
```

Comment:
Seems like a typo, should be "synced_except_target"
```suggestion
			.buildAndRegister(Identifier.of(MOD_ID, "synced_except_target"));
```